src/main/java/Duke.java

```diff
@@ -1,10 +1,184 @@
+import java.util.Scanner;
+
+
 public class Duke {
+    private static Task[] taskList = new Task[100];
+    private static String[] storeUserText = new String[100];
+    private static boolean doExit = false;
```

Comment:
Boolean names should sound like a boolean. For example, "isExitting", "isRunning", etc, instead of "doExit" which sounds like a method.